public/index.ejs

```diff
@@ -181,12 +185,26 @@ async function update() {
   if (username === '')
     return;
 
+  // Abort resources being fetched.
+  abortPendingFetches();
+
   // Get a list of all the user's repositories, and filter them based on whether they have deployments urls.
   let nextLink = `https://api.github.com/users/${username}/repos?per_page=50`;
 
   while (nextLink) {
     console.log(`Fetching nextLink=${nextLink}`);
-    let response = await fetch(nextLink, {headers: kGitHubHeaders});
+
+    // create a new AbortController and add it the list
+    const controller = new AbortController();
+    let abortControllerObject = {};
+    abortControllerObject[nextLink] = controller;
+    abortControllersList.push(abortControllerObject);
+
+    const options = {
+      headers: kGitHubHeaders,
+      signal: controller.signal
+    };
+    let response = await fetch(nextLink, options);
```

Comment:
Now that these fetches can become rejected due to the AC, we should add a rejection handler so these failed fetches don't turn into uncaught exceptions. Since we're using async/await that should be easy. Maybe something like this?

```suggestion
    let response = null;
    try {
      response = await fetch(nextLink, options);
    } catch(e) {
      console.assert(e.name === 'AbortError');
    }
    if (response === null) {
      continue;
    }
```